README.md

```diff
@@ -30,22 +45,39 @@ npm install messageformat-loader
 
 ### example.js
 
-``` javascript
+ES5, without configuration:
+```js
 var messages = require('messageformat-loader?locale=en!./messages.json');
 messages['ordinal-example']({ N: 1 });
 // => 'The 1st message.'
 ```
 
+ES6, with configuration:
```

Comment:
I'd put this one first. The ES5, no config way is nice because there's less setup, but if we're already going to show them the webpack config (since it gives a good overview of the options), then the ES6 way is simpler.